docs/roadmap/cosmos-hub-roadmap-2.0.md

```diff
@@ -52,39 +52,18 @@ The upgrades aim to add features such as liquidity, economic security, usability
 - Gaia v8.0.x
```

Comment:
Missing updates for ibc and packet-forward-middleware.

Reply:
Added